ArduPlane/Attitude.cpp

```diff
@@ -996,6 +1003,34 @@ void Plane::set_servos(void)
         channel_output_mixer(g.vtail_output, channel_pitch->radio_out, channel_rudder->radio_out);
     } else if (g.elevon_output != MIXING_DISABLED) {
         channel_output_mixer(g.elevon_output, channel_pitch->radio_out, channel_roll->radio_out);
+              //if (both) differential spoilers setup then apply rudder
```

Comment:
There is a bug in this code.  Because its calling set_radio it doesn't take into account if a channel is reversed i.e. RC5_REV=-1.  
It needs to be changed to call set_servo_out as that then calls calc_pwm which does take into account channel reversal.